manim/mobject/coordinate_systems.py

```diff
@@ -225,6 +225,42 @@ def get_axis_labels(
         )
         return self.axis_labels
 
+    def add_coordinates(
+        self, *axes_numbers: Optional[Iterable[float]], **kwargs
+    ) -> VGroup:
+        """Adds labels to the axes.
+
+        axes_numbers
+            The the numbers to be added to the axes. Use ``None`` to represent an axis without labels.
+
+        Examples
+        --------
+
+        .. code-block:: python
+
+            ax = ThreeDAxes()
+            x_labels = range(1, 11)
+            z_labels = range(2, 12, 2)
+            ax.add_coordinates(x_labels, None, z_labels)  # x & z labels
+            ax.add_coordinates(x_labels)  # only x labels
+
+        Returns
+        -------
+        VGroup
+            A :class:`VGroup` of the number mobjects.
+        """
+
+        self.coordinate_labels = VGroup()
+        # if nothing is passed to axes_numbers, produce axes with default labelling
+        if not axes_numbers:
+            axes_numbers = [None for x in self.axes]
```

Comment:
Since we aren't really using any value from the `self.axes` iterations...
```suggestion
            axes_numbers = [None for _ in range(self.dimension)]
```